### PR TITLE
Fix build warning for unused variable and mismatched tags.

### PR DESCRIPTION
**Pull request by zyfjeff**
Have many compile warnning for "defined as a struct here but previously declared as a class [-Wmismatched-tags]" and "unused variable"

### CODE REVIEW

**Comment by zyfjeff** [comment]
Fix incorrect email address

File: opencensus/stats/examples/exporter_example.cc [hunk 1]
```diff
@@ -24,7 +24,7 @@
 namespace opencensus {
 namespace stats {
 
-struct StatsExporterTest {
+class StatsExporterTest {
   static constexpr auto& ExportForTesting = StatsExporter::ExportForTesting;
```
Comment (g-easy) [inline opencensus/stats/examples/exporter_example.cc]: Not "public:" ?

**Comment by zyfjeff** [comment]
I'm sorry, I have updated the again

File: opencensus/stats/examples/exporter_example.cc (hunk 1 above)
Comment (zyfjeff) [reply opencensus/stats/examples/exporter_example.cc]: Done

File: opencensus/stats/internal/aggregation.cc [hunk 2]
```diff
@@ -33,6 +33,8 @@ std::string Aggregation::DebugString() const {
       return absl::StrCat("Distribution with ",
                           bucket_boundaries_.DebugString());
     }
+    default:
+      return std::string();
```
Comment (g-easy) [inline opencensus/stats/internal/aggregation.cc]: Please move this after the switch() statement. The reason is that providing a default: case suppresses warning about missing a case in an enum.

File: opencensus/stats/internal/aggregation_window.cc [hunk 3]
```diff
@@ -26,6 +26,8 @@ std::string AggregationWindow::DebugString() const {
     case Type::kInterval:
       return absl::StrCat("Interval (", absl::ToDoubleSeconds(duration_),
                           "s window)");
+    default:
+      return std::string();
```
Comment (g-easy) [inline opencensus/stats/internal/aggregation_window.cc]: Ditto.

File: opencensus/trace/internal/local_span_store_impl.cc [hunk 4]
```diff
@@ -127,7 +127,7 @@ std::vector<SpanData> LocalSpanStoreImpl::GetLatencySampledSpans(
         latency_ns < filter.upper_latency_ns) {
       out.emplace_back(span);
     }
-    if (out.size() >= filter.max_spans_to_return) break;
+    if (out.size() >= static_cast<size_t>(filter.max_spans_to_return)) break;
```
Comment (g-easy) [inline opencensus/trace/internal/local_span_store_impl.cc]: Please don't do this. These sign conversion warnings are fixed by https://github.com/census-instrumentation/opencensus-cpp/pull/11

File: opencensus/stats/internal/aggregation.cc (hunk 2 above)
Comment (zyfjeff) [reply opencensus/stats/internal/aggregation.cc]: Done

File: opencensus/stats/internal/aggregation_window.cc (hunk 3 above)
Comment (zyfjeff) [reply opencensus/stats/internal/aggregation_window.cc]: Done

File: opencensus/trace/internal/local_span_store_impl.cc (hunk 4 above)
Comment (zyfjeff) [reply opencensus/trace/internal/local_span_store_impl.cc]: Done

**Comment by g-easy** [comment]
Great, thanks!

**Pull request merged**